plugins/genesys/fu-genesys-usbhub-device.c

```diff
@@ -1234,6 +1342,10 @@ fu_genesys_usbhub_device_attach(FuDevice *device, FuProgress *progress, GError *
 	if (!fu_genesys_usbhub_device_reset(self, error))
 		return FALSE;
 
+	if (self->hid_channel != NULL) {
+		fu_device_add_flag(FU_DEVICE(self->hid_channel), FWUPD_DEVICE_FLAG_WAIT_FOR_REPLUG);
+	}
+
 	/* success */
 	fu_device_add_flag(device, FWUPD_DEVICE_FLAG_WAIT_FOR_REPLUG);
```

Comment:
shouldn't this be or? as in put the `fu_device_add_flag(device, FWUPD_DEVICE_FLAG_WAIT_FOR_REPLUG);` in an `else`

Reply:
The HID is the substitute of hub; after sent reset by hid, hub also reset.